htdocs/product/inventory/inventory.php

```diff
@@ -88,6 +88,11 @@
 	$action = 'view';
 }
 
+// Prevent updating lines when not posted by SAVE input button
+if (!GETPOSTISSET('save') && $action == 'updateinventorylines') {
```

Comment:
So this means that:
You edit the first page of inventory by changing quantities, you switch tonthe next page to edit the quantities for next page and you loose all your change of previous page if you didn't click the save before changing page ?

Reply:
No, it is not the problem. And btw my PR is not the best answer to it - it corrects the bug but have to click on save button each time before changing page, which i thought was the normal behaviour :(

I close this PR and submitted another: #24709